src/aerovaldb/sqlitedb/sqlitedb.py

```diff
@@ -436,7 +436,10 @@ async def _get(self, route, route_args, *args, **kwargs):
         raise UnsupportedOperation
 
     async def _put(self, obj, route, route_args, *args, **kwargs):
-        assert len(args) == 0
+        if len(args) > 0:
+            raise UnusedArguments(
+                f"Unexpected positional arguments {args}. Jsondb does not use additional positional arguments currently."
+            )
```

Comment:
Not sure why this is here? A user should transparently switch between one implementation and another, so *args and *kwargs can only be optional arguments. They should be silently ignored by other implementations.

Or, if we don't have any use-case for *args, maybe remove it completely?

Reply:
And the code seems to be a bad duplicate, since it mentions Jsondb in the sqlitedb implementation.

Reply:
Agreed. I did this to mask confusing error messages if too many arguments are provided.

I think it will be necessary to still raise an exception, to provide a useful error message, but it should probably be in the get/put decorators.